nengo_gui/guibackend.py

```diff
@@ -295,7 +295,7 @@ def get_session(self):
         return session
 
     def log_message(self, format, *args):
-        logger.info(format, *args)
+        logger.debug(format, *args)
```

Comment:
What are “these” log messages? It seems that this function isn't called in normal operation. Maybe in some specific error cases? But if it is error cases a higher log level might be better?